src/Backend/Form/DataTransferObject/CheckboxEnabledFieldDataTransferObject.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Backend\Form\DataTransferObject;
+
+class CheckboxEnabledFieldDataTransferObject
+{
+    public bool $enableField = false;
+
+    public string $field = '';
+
+    public static function create(bool $enableField, string $field): self
```

Comment:
Not a fan of the naming here, field makes me think it is the name of the form field instead of the value.
I would suggest using `enabled` and `value` instead.
Also I would drop the string type and changed it to mixed so that we can use it for datefields etc as well